gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages.properties

```diff
@@ -302,6 +302,7 @@ edit_assignment_save_percentage=You have successfully updated gradebook item {0}
 edit_assignment_save_changes=Save Changes
 edit_assignment_external_items= * Please go to {0} to edit these settings.
 edit_assignment_due_date_external=Due Date *
+edit_assignment_error_title_assignment=It's not possible modify this gradebook item's name because some assignment sends notes here. Assignment:
```

Comment:
"notes" might be inconsistent in the context of Messages.properties (en_US). Should it read "grades"?